api/interestr/api/serializers.py

```diff
@@ -18,23 +18,32 @@ class Meta:
 class GroupSerializer(serializers.ModelSerializer):
     data_templates = DataTemplateSerializer(many=True, read_only=True)
     tags = TagSerializer(many=True, read_only=True)
+    members = serializers.PrimaryKeyRelatedField(many=True, read_only=True)
+    moderators = serializers.PrimaryKeyRelatedField(many=True, read_only=True)
 
     class Meta:
         model = core_models.Group
-        fields = ('id', 'name', 'created', 'updated', 'size', 'members', 'moderators', 'picture', 'data_templates',
+        fields = ('id', 'name', 'created', 'updated', 'size', 'members', 'location', 'description', 'moderators', 'picture', 'data_templates',
                   'tags', )
+class CommentSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = core_models.Comment
+        fields = ('id', 'owner', 'text', 'post', 'created', 'updated',)
 
 class UserSerializer(serializers.ModelSerializer):
     joined_groups = GroupSerializer(many=True, read_only=True)
     moderated_groups = GroupSerializer(many=True, read_only=True)
     data_templates = DataTemplateSerializer(many=True, read_only=True)
+    comments = CommentSerializer(many=True,) 
```

Comment:
Is this really works? I think the trailing comma describes a tuple. It should not be used with function calls/constructors.

Reply:
I was testing whether it would work, then forgot to change it. Even though I think that it works, there is no reason not to make it read-only.

Reply:
It's ok. We have pagination working.